src/DynamicData/List/ChangeAwareList.cs

```diff
@@ -84,7 +84,7 @@ public IChangeSet<T> CaptureChanges()
                     return ChangeSet<T>.Empty;
                 }
 
-                returnValue = new ChangeSet<T>(_changes);
+                returnValue = _changes;
```

Comment:
We can now fill a changeset up directly, and pass it as the return value when we want a capture.  We're always doing a clear afterwards, so downstream users will not have reference to our internal changeset after this call.